src/controllers/stallController.ts

```diff
@@ -244,17 +244,66 @@ async function importStalls(req: Request, res: Response, next: NextFunction) {
     let parseError = '';
     let currRow = 2; // header is on first row
 
+    type stallRow = {
```

Comment:
Should use `StallCreationAttributes` from `stall.ts` so we don't maintain two copies of the same attributes

Reply:
ok I removed transformedStallRow instead bc stallRow's openingHours is string rather than JSON

Reply:
Can remove this portion here if the modification below is done.

Reply:
ok